doc/sphinx/language/coq-library.rst

```diff
@@ -989,6 +989,17 @@ Notation    Interpretation  Precedence  Associativity
 Floats library
 ~~~~~~~~~~~~~~
 
+The standard library offers a small ``Floats`` module to give access
+to the processor floating-point operators through the kernel of Coq.
+The main use of this floating-point arithmetic is probably to perform
+proofs by reflection. Note however that, while the standard library
+offers the computation capability and a bit-level specification, it
+doesn't include any elaborate theorems, such as a link with real
+arithmetic or various error bounds. This makes the module alone pretty
+useless for the proof part of most proofs by reflection. A much more
+expansive set of results can be found in the
+`Flocq <https://flocq.gitlabpages.inria.fr/>`_ library.
```

Comment:
```suggestion
The standard library has a small ``Floats`` module for accessing
processor floating-point operations through the Coq kernel.
However, while this module supports
computation and has a bit-level specification, it
doesn't include elaborate theorems, such as a link to real
arithmetic or various error bounds. This makes the module relatively
useless for proofs by reflection.
The `Flocq <https://flocq.gitlabpages.inria.fr/>`_ library is
a better choice for such proofs because it
provides a much more complete set of results.
```

"The main use of this floating-point arithmetic is probably to perform
proofs by reflection." seems to contradict "This makes the module alone pretty
useless for the proof part of most proofs by reflection."  

"pretty ..." - a bit too casual for doc

Reply:
You misunderstood. The library was indeed designed for proof by reflections (and "probably" could be dropped, there is nothing probabilistic about it). It is just that this goal is difficult to achieve when using just the standard library; "alone" was the key word here.

Reply:
Thanks, I reworded it a bit to preserve the original meaning. In particular, as Guillaume points out, Flocq doesn't come as a replacement for Floats, rather as a complement. I also removed the word "complete" which may induce, as already discussed above, that the bit-level spec in Floats isn't complete, which is wrong.